Makefile.main

```diff
@@ -204,10 +209,15 @@ no-changes-in-commit:
 		exit 2; \
 	fi
 
+export POSTGRESQL_VERSION RABBITMQ_VERSION
+ci-install:
+	$(CI_PATH)/scripts/install_services.sh
+
 .PHONY: dependencies $(DEPENDENCIES) \
 		build $(COMMANDS) \
 		test test-race test-coverage \
 		docker-login docker-validate docker-build docker-push \
 		packages \
 		clean \
-		no-changes-in-commit
+		no-changes-in-commit \
+		ci-install
```

Comment:
not to be addressed in this PR, but I'd like to propose to declare phony targets in place as a common recommendation.
I opened an issue https://github.com/src-d/ci/issues/77 to consider it.

Reply:
Makes sense, just a few characters more, but better context and less merge conflicts :+1: